distributed/diagnostics/__init__.py

```diff
@@ -1,11 +1,2 @@
-from contextlib import suppress
-
 from .graph_layout import GraphLayout
 from .plugin import SchedulerPlugin
-
-with suppress(ImportError):
-    from .progressbar import progress
```

Comment:
This is a circular dependency with distributed.client and will ALWAYS fail to import.
There is also a name collision with the module distributed.diagnostics.progress.
Anybody who, today, references distributed.diagnostics.progress is certainly referring to the module.